tests/features/taxonomy_browser.feature

```diff
@@ -0,0 +1,78 @@
+@api
+Feature: Taxonomy browser
+  In order to test ‘Taxonomy browser’ feature
+  As an administrator
+  I want to make sure ‘Taxonomy browser’ allows creating blocks with taxonomy vocabularies and filter by terms
+
+  Background:
+    Given I am logged in as a user with the 'administrator' role
+
+  @api
+  Scenario: Administrator user creates a new view to show the taxonomy block.
+    Given a content view with machine name "view_articles" is available
+    When I go to "/admin/structure/views/view/view_articles"
+    Then the response status code should be 200
+    Then I should see the text "Master details"
+    #Page display
+    And I press the "edit-displays-top-add-display-page" button
+    Then I should see the text "Page details"
+    When I go to "/admin/structure/views/nojs/display/view_articles/page_1/path_en"
+    Then I should see the text "Page: The menu path or URL of this view"
+    And I fill in "edit-path" with "/view-articles"
+    And I press the "Apply" button
+    Then I should see the text "* All changes are stored temporarily. Click Save to make your changes permanent. Click Cancel to discard your changes."
+    #Configure contextual
+    When I go to "/admin/structure/views/nojs/add-item/view_articles/default/argument_en"
+    Then I check "Content: Has taxonomy term ID (with depth)"
+    And I press the "Add and configure contextual filters" button
+    When I select the radio button "Provide default value"
+    And I fill in "Fixed value" with "fixed_value"
+    Then I check "Override title"
+    And I fill in "edit-options-title" with "%1"
+    And I press the "Apply" button
+    #Configure header
+    When I go to "/admin/structure/views/nojs/add-item/view_articles/default/header_en"
+    Then I check "edit-name-viewsview"
+    And I press the "Add and configure header" button
+    And I select "View: taxonomy_browser_header - Display: default" from "View to insert"
+    Then I check "Inherit contextual filters"
+    And I press the "Apply" button
+    When I go to "/admin/structure/views/nojs/add-item/view_articles/default/field_en"
+    Then I check "edit-name-nodetitle"
+    And I press the "Add and configure fields" button
+    And I press the "Apply" button
+    Then I should see the text "* All changes are stored temporarily. Click Save to make your changes permanent. Click Cancel to discard your changes."
+    And I press the "Save" button
+    Then I should see the text "has been saved."
+    #Add the block to a region
+    Given I visit "/admin/config/taxonomy_browser/settings_en"
+    And the checkbox "edit-taxonomy-browser-vocabulary-2" is checked
+    When I visit "/admin/structure/block_en"
+    Then I should see the text "taxonomy_browser: Tags"
+    When I go to "/admin/structure/block/manage/taxonomy_browser/taxonomy_browser_vocabulary_2/configure_en"
+    When I select "Sidebar Left" from "Ec_resp (default theme)"
```

Comment:
You should add the tag @theme_wip to the scenario because it requires a region and this will cause issues with the ec_europa theme

Reply:
Done! Thank you Imanol.